js/ui/calendar/ui.calendar.base_view.js

```diff
@@ -111,6 +117,11 @@ const BaseView = Widget.inherit({
             'append';
     },
 
+    _shouldRenderWeekColumn: function() {
+        const { showWeekNumbers, zoomLevel } = this.option();
+        return showWeekNumbers && zoomLevel === 'month';
```

Comment:
zoomLevel === 'month' -> ZOOM_LEVEL.MONTH

Reply:
there is no ZOOM_LEVEL object in base_view file, only in ui.calendar.js. And it's the only place where we check zoomLevel in base view, so I see no point to add this const here

Reply:
May be it should shared const variable in common files between the files it is used? Or can we import this const from file where it is declared?